src/containers/category_filter.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
`category_filter.js` is a "presentational" component, and according to this project's specification, presentational-components should be in the folder `"src/components/"` folder.

### please relocate this file

Reply:
All done.👍